test/compilable/traits.d

```diff
@@ -20,6 +20,7 @@ static assert(is(typeof(__traits(getTargetInfo, "cppRuntimeLibrary")) == string)
 version (CppRuntime_Microsoft)
 {
     static assert(__traits(getTargetInfo, "cppRuntimeLibrary") == "libcmt" ||
+                  __traits(getTargetInfo, "cppRuntimeLibrary").length == 0 ||
```

Comment:
Because I disabled the compiler demanding `-msvcrtlib` switch to do a Windows cross compile on Linux. `-msvcrtlib` is only needed for linking, not compiling or generating a library.

Reply:
I decided to defer the -mscrtlib switch problem, and removed that line.